### PR TITLE
Bug 1128438 - [Stingray][demo] add close button to fling sender app for ...

### PR DESCRIPTION
**Pull request by huchengtw-moz**
...testing

### CODE REVIEW

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c58e7079b3d5)

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=567a63466de7)

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=69e7c0c1268a)

File: tv_apps/fling-sender/js/fling_sender.js [hunk 1]
```diff
@@ -63,13 +67,20 @@
...
   proto.handleMessage = function fs_handleMessage(element, msg) {
-    element.textContent += JSON.stringify(msg);
+    element.textContent = JSON.stringify(msg);
   };
 
-  proto.enableButtons = function fs_enableButtons() {
-    ['load-btn', 'play-btn', 'pause-btn', 'seek-btn',
+  proto.handleStateChange = function fs_handleStateChange() {
+    if (!this._session || this._session.state === 'disconnected') {
+      this.enableButtons(true);
```
Comment (rexboy7) [inline tv_apps/fling-sender/js/fling_sender.js]: This reads somewhat weird.. I'd suggest change the function name to `disableButtons` or invert the argument such that the argument specifies whether it "enables" the button or not.

Comment (huchengtw-moz) [inline tv_apps/fling-sender/js/fling_sender.js]: ha..sure.

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=96a45bff32e0)

**Pull request merged**